### PR TITLE
[PS-1453] add tracking sdk to productcat-sdk-javascript as a dependency 

### PR DESCRIPTION
**Pull request by swVisenze**

### CODE REVIEW

File: examples/productsummary_url.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
   <script type="text/javascript" src="js/snippet.js"></script>
   <script type="text/javascript">
     // TODO: insert your app key here
-    productcat.set('app_key', 'APP_KEY');
-    productcat.set('cid', "CID")
+    productcat.set('app_key', 'aa8d950812fc16938322f2ba78a1ac53');
```
Comment (thehung111) [inline examples/productsummary_url.html:11]: can remove this and clean history?

File: README.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ you can also set other parameters with the productcat.set method, like country,
 ```js
 productcat.set('app_key', 'YOUR_APP_KEY'); // required.
 productcat.set('cid', "YOUR_CID"); // required.
+productcat.set('tracker_code', 'test')  // code for visenze tracking
```
Comment (thehung111) [inline README.md:49]: @swVisenze actually it is not important to migrate tracking to ProductCat SDK. What is important is to update ViSearch Javascript SDK (need to sync with Dashboard release for customers to get the tracking code). For ProductCat, we probably will hold this first as we already have an existing data pipeline.

File: js/productcat.js [hunk 3]
```diff
@@ -195,6 +200,26 @@
...
 
+  const sendEvent = (path, response) => {
+    if(settings.tracker_code) {
+      if(!tracker){ 
+        console.log("init tracker");
+        console.log("uid: ", settings.uid)
+        tracker = va.init({code: settings.tracker_code, uid: settings.uid})
+      }      
+
+      if(path === "summary/products" || path === "summary/products/srp" || path === "mps/products") {
```
Comment (thehung111) [inline js/productcat.js:211]: no need to do this. We will get search event directly from our log instead.

**Review by swVisenze — COMMENTED** [review]
try delete

**Comment by swVisenze** [comment]
close

**Pull request closed without merging**